feature/tabsflow/moreflow/about/src/main/java/com/epmedu/animeal/tabs/more/about/AboutScreenState.kt

```diff
@@ -0,0 +1,7 @@
+package com.epmedu.animeal.tabs.more.about
```

Comment:
State classes should also be in viewmodel sub-package

Reply:
changes added